python/py-mdtraj/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        mdtraj mdtraj 1.9.0
+name                py-mdtraj
+homepage            http://www.mdtraj.org
+platforms           darwin
+license             LGPL-2.1+
+maintainers         {gmail.com:giovanni.bussi @GiovanniBussi} openmaintainer
+
+description         A modern, open library for the analysis of molecular dynamics trajectories
+long_description    Read, write and analyze MD trajectories with only a few lines of Python code. \
+                    For details, see the website at mdtraj.org. MDTraj is research software. \
+                    If you make use of MDTraj in scientific publications, please cite it.
+
+supported_archs     i386 x86_64
+
+checksums           rmd160  58a3684fdf7a81813699aa281c411c04b130b91b \
+                    sha256  e93db244035b7551488cea24d96dd813e67ce655461fc50f53c4d8a9cd8bd7ca
+
+python.versions     27 34 35 36
+
+if {${name} ne ${subport}} {
```

Comment:
Why this bit? There's no subports here, right? Maybe I'm confused?

Reply:
@pmetzger I did not find any documentation, but if you look in the python directory all ports look like this.

My interpretation is that there is a common port (`py-mdtraj`), which implicitly depends on the default subport (`py27-mdtraj`). In practice, if I remove the if statement I get the following error
````
Error: Unable to open port: can't read "python.version": no such variable
````
since within the main port (`py-mdtraj`) python version is not known. I guess this is the expected behavior.

Thanks!

Reply:
I've never done a python package port, so I didn't realize that. My bad. I think this is therefore all fine.